keel-core/src/main/kotlin/com/netflix/spinnaker/keel/actuation/ResourceActuator.kt

```diff
@@ -265,7 +265,11 @@ class ResourceActuator(
     }
 
   private fun DeliveryConfig.environmentFor(resource: Resource<*>): Environment? =
-    environments.firstOrNull { it.resources.contains(resource) }
+    environments.firstOrNull {
```

Comment:
should we add a log message here as well?

Reply:
We're already logging when this method fails, it's just happening elsewhere in the code: this call is wrapped in a checkNotNull on line 91, which throws an exception that logs the error in the catch block.